template.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Codice Website </title>
+	<link rel="stylesheet" type="text/css" href="bootstrap/css/bootstrap.css">
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<link href="https://fonts.googleapis.com/css?family=Muli:300,400,600" rel="stylesheet">
+	<link rel="shortcut icon" type="image/png" href="images/favicon.png">
+</head>
+
+<body>
+<div class="shadow-wrapper">
+<div class="content-wrap">
+	
+	<div class="container">
+		<div class="header-wrap-pages header-main row">
+			<div class="logo col-md-4">
+				<a href="index.html"><img src="images/codice-logo.png" /></a>
+			</div>
+
+			<div class="col-md-8">
+				<nav>
+					<div class="nav-item"><a href="#">About</a>
+						<ul class="sub-menu">
+							<li><a href="about.html">About Codice</a></li>
+							<li><a href="governance.html">Governance</a></li>
+							<li><a href="board.html">The Board</a></li>
+							<li><a href="roles.html">Roles</a></li>
+							<li><a href="bylaws.html">Bylaws</a></li>
+						</ul>
+					</div>
+					<div class="nav-item"><a href="#">Contributing</a>
+						<ul class="sub-menu">
+							<li><a href="contributing.html">How To Contribute</a></li>
+							<li><a href="agreements.html">Agreements</a></li>
+							<li><a href="proposal.html">Proposal Templates</a></li>
+							<li><a href="incubation.html">Incubation</a></li>
+							<li><a href="graduation.html">Incubation Graduation</a></li>
+						</ul>
+					</div>
+					<div class="nav-item"><a href="tools.html">Tools</a></div>
+				</nav>
+			</div>
+		</div>
+	</div>
+
+	<div class="container">
+		<div class="main-content-wrap">
+				This is where content goes.
```

Comment:
It's not used in the actual website, so it doesn't matter. It's just there as a bare bones template. Whatever is in there will be deleted anyway.